docs/xxusegcstartuphints.md

```diff
@@ -50,7 +50,7 @@ The final value stored to the shared cache is not recorded in the verbose GC out
 
 :fontawesome-solid-pencil:{: .note aria-hidden="true"} **Notes:**
 
-- When enabled, this option overrides any initial heap size that is specified on the command line, for example by using the [`-Xms`](xms.md) option.
+- When enabled, this option overrides any initial heap size that is specified on the command line but only if the new size is within the initial heap size limits of that command-line option. For example, the new initial heap size will be greater than the initial heap size set by the [`-Xms`]`(https://eclipse.dev/openj9/docs/xms/) option or its default value.
```

Comment:
The link isn't working, there appears to be an extra back-tick.
[`-Xms`]`(https://eclipse.dev/openj9/docs/xms/)